Expensify/App
# Added Workspace Menu

cc - @chiragsalian 

### Details

- Adds a generic popup menu to Header with options
- Uses this popup menu to add "New workspace" and "Delete Workspace" options

### Fixed Issues

$ https://github.com/Expensify/App/issues/5185

### Tests
1. Tested the popup for all platforms - including hiding close button on desktop
2. Checked the options for New Workspace
3. Changed by updating language and translations
4. Pending: Delete Workspace Testing


### QA Steps
1. Go to the right sidebar
2. Click on any Workspace item
3. You should see three dots (more options) menu, hover over it
4. You should see two options - New Workspace and Delete Workspace
5. Test both the actions are working fine
6. Try the same steps for mobile, except that the popup should be docked at the bottom


### Tested On


### Screenshots


#### Web

<img width="506" alt="web-workspace-menu" src="https://user-images.githubusercontent.com/3069065/140766676-f2346646-5320-4001-8399-7ac42ca0388e.png">

#### Mobile Web

<img width="337" alt="mweb-workspace-menu" src="https://user-images.githubusercontent.com/3069065/140766701-e2c7fe91-b023-47de-8794-21ec4a43f376.png">

#### Desktop

<img width="470" alt="desktop-workspace-menu" src="https://user-images.githubusercontent.com/3069065/140766746-337b5fc4-d96d-4176-bedd-fb84f9319029.png">

#### iOS

<img width="378" alt="ios-workspace-menu" src="https://user-images.githubusercontent.com/3069065/140770830-9925d9be-b877-48b5-8e7e-12e3923067f8.png">

#### Android

<img width="311" alt="android-workspace-menu" src="https://user-images.githubusercontent.com/3069065/140769377-078c27ff-4795-4c48-a572-e192ff73bafd.png">
